server/app/models/Version.java

```diff
@@ -171,4 +179,29 @@ public boolean removeTombstoneForQuestion(Question question) {
     }
     return this.tombstonedQuestionNames.remove(question.getQuestionDefinition().getName());
   }
+
+  @VisibleForTesting
+  public boolean addTombstoneForProgramForTest(Program program) {
```

Comment:
suggestion (blocking): please document this method's return type

Reply:
Done for this and oher existing undocumented methods. Attempted to keep it consistent with the language used in `program/questionIsTombstoned`.